templates/base.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}Taxi Service{% endblock %}</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static "css/styles.css" %}">
+</head>
+<body>
+     <div class="container">
+        <div class="sidebar">
+            {% include 'includes/sidebar.html' %}
+        </div>
+
+        <div class="content">
+            {% block content %}
+                <p>Default Content</p>
+            {% endblock %}
+        </div>
+     </div>
+</body>
+</html>
+
```

Comment:
It's a good practice to end HTML files with a blank line for better formatting and to avoid potential issues with some version control systems.